_gcode/G000-G001.md

```diff
@@ -51,7 +51,7 @@ parameters:
 
 - tag: F
   optional: true
-  description: The maximum movement rate of the move between the start and end point. The feedrate set here applies to subsequent moves that omit this parameter.
+  description: The maximum movement rate (in mm/min) of the move between the start and end point. The feedrate set here applies to subsequent moves that omit this parameter.
```

Comment:
If the current units are set to inches, then `F` indicates a feedrate in inches-per-minute.